gitbooks/developing/architecture/frontend.zh-CN.md

```
@@ -0,0 +1,2295 @@
+| [Hook 与工具](frontend.zh-CN.md#hooks-utilities) | 共享 hook、辅助函数、配置 |
+
+## 规模
+
+| 指标 | 值 |
+| --------------------------------------- | ------------------------------------------------------------------------ |
+| `app/src/` 下的 TypeScript / TSX 文件 | \~285 (`find app/src -name '*.ts' -o -name '*.tsx' \| wc -l` 刷新) |
+| 测试 runner | Vitest (`app/test/vitest.config.ts`) |
+
+## 目录布局
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)